pages/test-evaluate/evaluation-tools/selecting-eval-tools.pl.md

```diff
@@ -0,0 +1,230 @@
+
+**Oceniany produkt**
+: Zależnie od rodzaju produktu, który oceniasz (np. strona internetowa, dokument, aplikacja), możesz potrzebować konkretnego narzędzia, które będzie Ci najbardziej pomocne. Niektóre narzędzia są przystosowane do oceny stron internetowych, inne do dokumentów, aplikacji komputerowych, aplikacji mobilnych itp.
+
+**Płatne lub bezpłatne**
+: Narzędzia oceny są dostępne w ramach różnych typów licencji, takich jak licencja Open Source, komercyjna, korporacyjna itp. Niektóre narzędzia są płatne, inne darmowe.
+
+**Zasada działania**
+: Narzędziami oceny mogą być wtyczki (rozszerzenia) do narzędzi autorskich i przeglądarek internetowych, narzędzia wiersza poleceń, aplikacje komputerowe lub mobilne oraz usługi internetowe. W zależności od procesu projektowania i tworzenia można łączyć różne narzędzia.
+
+**Standardy**
+: Różne organizacje i rządy mogą wymagać zgodności z różnymi standardami dostępności, dlatego różne narzędzia obsługują te standardy. Wytyczne W3C dla dostępności treści internetowych (WCAG) są uznawane na arenie międzynarodowej za standard dostępności stron internetowych i obsługiwane przez większość narzędzi.
+
+**Typ pliku**
+: Większość narzędzi oceny sprawdza dostępność treści HTML. Niektóre sprawdzają inne technologie internetowe, takie jak EPUB, WAI-ARIA, CSS, SVG i PDF.
+
+**Zakres**
+: Zakres tego, co narzędzie oceny może automatycznie testować, różni się w zależności od narzędzia. Niektóre narzędzia sprawdzają pojedynczą stronę, inne natomiast sprawdzają całe grupy powiązanych stron. Niektóre mogą również uzyskać dostęp do treści chronionych hasłem.
+
+**System operacyjny**
+: Niektóre narzędzia działają tylko w określonych systemach operacyjnych, takich jak Windows, MacOS, Linux, Android, iOS.
+
+**Przeglądarka**
+: Niekiedy narzędzia działają tylko w określonych przeglądarkach, takich jak Chrome, Edge, Safari, Firefox itp.
+
+**Deklaracja dostępności**
+: Ważne jest, aby narzędzia oceny były dostępne, aby mogły z nich korzystać osoby z niepełnosprawnościami. Niektórzy dostawcy podają informacje na temat dostępności swoich narzędzi w deklaracji dostępności.
+
+**Zasady ACT**
+: Dostawcy narzędzi mogą opisać, czy i w jaki sposób ich narzędzia obsługują zasady testowania zgodności dostępności. Jeżeli istnieje raport dotyczący wdrożenia zasad, w sekcji szczegółów narzędzia na Liście narzędzi oceny dostępności cyfrowej będzie podane łącze do raportu. Więcej informacji można znaleźć w artykule [[Wdrażanie zasad ACT w narzędziach i metodologiach oceny]](https://www.w3.org/WAI/standards-guidelines/act/implementations/).
+
+**Język**
+: Narzędzia oceny obsługują różne języki. Obejmuje to interfejs użytkownika narzędzia, a także język obsługiwanych treści (na przykład w celu wykrycia barier językowych).
+
+**Wsparcie dla**
+: Niektóre narzędzia oceny dostępności cyfrowej mogą wygenerować pełny raport dostępności. Możesz filtrować, aby znaleźć narzędzia, które mogą Ci pomóc:
```

Comment:
@zwiastunsw The original version reads "Some web accessibility evaluation tools can **(help you)** generate a complete accessibility report."

Could you adapt the translation so that the user understands: some tools can generate or help you generate?